docs/user/azure/install.md

```diff
@@ -84,32 +40,6 @@ INFO Access the OpenShift web-console here: https://console-openshift-console.ap
 INFO Login to the console with user: kubeadmin, password: 5char-5char-5char-5char
 ```
 
-Note that Azure support is still in development and as a result, certificate signing requests (CSRs) for compute nodes will need to be manually approved before the cluster will complete installation. The list of CSRs can be viewed using the `oc` client tool:
```

Comment:
When was this fixed?

Reply:
I don't know when this was fixed. This has not been necessary since I've been working on Azure. I did ask @abhinavdahiya about nuking this section and was given the thumbs up, so he might know when it happened.